nautobot/extras/filters/customfields.py

```diff
@@ -71,14 +72,23 @@ class CustomFieldJSONFilter(CustomFieldFilterMixin, django_filters.Filter):
     """Custom field single value filter for backwards compatibility"""
 
 
-class CustomFieldMultiSelectFilter(CustomFieldFilterMixin, django_filters.Filter):
-    """Custom field single value filter for backwards compatibility"""
+class CustomFieldMultiSelectFilter(CustomFieldFilterMixin, MultiValueCharFilter):
+    """This provides functionality for filtering custom fields with multiple  select type"""
 
     def __init__(self, *args, **kwargs):
         kwargs.setdefault("lookup_expr", "contains")
         super().__init__(*args, **kwargs)
 
 
+class CustomFieldMultiValueSelectFilter(CustomFieldFilterMixin, django_filters.MultipleChoiceFilter):
+    """This provides functionality for filtering custom fields with select type"""
+
+    def __init__(self, *args, **kwargs):
+        # TODO(jathan): See if I can make this user `kwargs` instead.
```

Comment:
I think we can ditch this comment. As far as I can tell there is no way to overload the widget more directly except on a FilterSet class.